drivers/hmis/app/models/hmis/form/form_processor.rb

```diff
@@ -81,6 +81,11 @@ def owner_factory(create: true) # rubocop:disable Lint/UnusedMethodArgument
     owner
   end
 
+  def current_living_situation_factory(create: true)
+    owner.information_date = Date.today if create
+    owner
+  end
+
```

Comment:
CLS is a bit tricky because I anticipate that it can be collected as its own form (which is what youre doing here) OR collected as part of a Custom Assessment. To prepare for that assessment scenario, this needs to be a bit more complicated:

```suggestion
  def current_living_situation_factory(create: true)
    # If this is a form just for collecting CLS, it is the owner
    return owner if owner.is_a? Hmis::Hud::CurrentLivingSituation

    # If this is an assessment, CLS may already exist in relationship to the FormProcessor
    return current_living_situation if current_living_situation.present? || !create

    # If not, create a new CLS
    self.current_living_situation = enrollment_factory.current_living_situations.build(**common_attributes)
  end
  
```